### PR TITLE
fix some shellcheck/stickler compaints in debug script

### PR DESCRIPTION
**Pull request by PromoFaux**
**By submitting this pull request, I confirm the following:**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#git-commit---signoff))

---

**What does this PR aim to accomplish?:**

Fixing shellcheck issues in the debug script

**How does this PR accomplish the above?:**

**What documentation changes (if any) are needed to support this PR?:**

### CODE REVIEW

File: advanced/Scripts/piholeDebug.sh [hunk 1]
```diff
@@ -115,53 +116,52 @@ PIHOLE_WEB_SERVER_ERROR_LOG_FILE="${WEB_SERVER_LOG_DIRECTORY}/error.log"
...
-${PIHOLE_SCRIPTS_DIRECTORY}
-${BIN_DIRECTORY}
-${RUN_DIRECTORY}
-${LOG_DIRECTORY}
-${WEB_SERVER_LOG_DIRECTORY}
-${WEB_SERVER_CONFIG_DIRECTORY}
-${HTML_DIRECTORY}
-${WEB_GIT_DIRECTORY}
-${BLOCK_PAGE_DIRECTORY})
+#REQUIRED_DIRECTORIES=("${CORE_GIT_DIRECTORY}"
```
[the remark this answers is not in the document]
Comment (PromoFaux) [reply advanced/Scripts/piholeDebug.sh]: Ah, yeah I meant to ask @jacobsalmela to review the ones I'd commented out to make sure I hadn't overlooked their "neededness"

[the remark this answers is not in the document]
Comment (dschaper) [reply advanced/Scripts/piholeDebug.sh]: Perfect! Thanks!

[the remark this answers is not in the document]
Comment (jacobsalmela) [reply advanced/Scripts/piholeDebug.sh]: I think I added some vars I never got around to using so they can be kept in for future development or removed completely.  I'm find with it either way.

**Comment by dschaper** [comment]
I'm actually fine with them being left in for now and get this one merged. We can pull out the extras later on when I'm revising the shell code.

**Comment by jacobsalmela** [comment]
Sounds good.  I was planning ahead for some future ideas, but just never went back to implement them.  Thanks for fixing this stuff @PromoFaux

**Pull request merged**